examples/porting/lv_port_disp_template.c

```diff
@@ -132,20 +142,38 @@ static void disp_init(void)
     /*You code here*/
 }
 
+volatile bool disp_flush_enabled = true;
+
+/* Enable updating the screen (the flushing process) when disp_flush() is called by LVGL
+ */
+void disp_enable_update(void)
+{
+    disp_flush_enabled = true;
+}
+
+/* Disable the flushing process when disp_flush() is called by LVGL
+ */
+void disp_disable_update(void)
```

Comment:
This comment and the same in C file are intentionally not updated?

Reply:
Sorry, I don't understand.

Reply:
I mean the comment of the `disp_disable_update` is
```
Disable the flushing process when disp_flush() is called by LVGL
```

But for `disp_enable_update`:
```
Enable updating the screen (the flushing process) when disp_flush() is called by LVGL
```

So they have a different format. 

Sorry for the nitpicking.